modules/n1ql/pages/n1ql-language-reference/delete.adoc

```diff
@@ -3,7 +3,7 @@
 :page-topic-type: reference
 :imagesdir: ../../assets/images
 
-:authorization-overview: xref:learn:security/authorization-overview.adoc
+:authorization-overview: xref:server:learn:security/authorization-overview.adoc
```

Comment:
I can see why you've done this, but I'm not sure if this change is necessary. As this is the release/7.6 branch, we're already in the server component. We would add the server component to this xref when this change is merged into Capella. Not going to insist that you revert this but it's got me thinking 🤔

Reply:
links are broken in a local build without, I won't test them going forward